packages/inference/src/snippets/templates/js/fetch/textToImage.jinja

```diff
@@ -0,0 +1,21 @@
+{% if provider == "hf-inference" %}
```

Comment:
What's the reason for not outputing a snippet when the provider is external?

Reply:
hmm, don't remember the reason :thinking: I reused what was existing before (see https://huggingface.co/black-forest-labs/FLUX.1-schnell?inference_api=true&inference_provider=hf-inference&language=js vs https://huggingface.co/black-forest-labs/FLUX.1-schnell?inference_api=true&inference_provider=together&language=js).

But now that we use `makeRequestOptions` it shouldn't be an issue anymore.

Reply:
fixed in https://github.com/huggingface/huggingface.js/pull/1291/commits/18d87cd6f313dfcfead060521546d11a797bd1a6